docs/docs/mlflow-3/index.mdx

```diff
@@ -0,0 +1,182 @@
+In deep learning, training often generates multiple model checkpoints, where the best candidates are further evaluated before production deployment. 
+
+We are introducing a new first-class object, the <APILink fn="mlflow.entities.LoggedModel">LoggedModel</APILink> entity, into MLflow Tracking to streamline these processes. As you define and evaluate your GenAI agents in code, 
+or your deep learning jobs create and evaluate models, they will be automatically stored as MLflow Logged Models in your MLflow Experiment. 
+
+Each GenAI agent consolidates its evaluation metrics from both offline jobs and online interactions if you use managed MLflow with Databricks. 
+This feature removes the need to query Runs for traces from different environments (batch jobs, different notebook sessions etc.) to gain a full understanding of the agent’s performance.
+Similarly, every deep learning checkpoint is stored as a <APILink fn="mlflow.entities.LoggedModel">LoggedModel</APILink> with direct links to its own metrics. There is no longer any need to carefully log evaluation metrics for each checkpoint
+at its epoch all within the same Run, or to later query all metrics at a certain epoch just to retrieve metrics for one checkpoint. 
+
+Specifically, you will be able to:
+- View Logged Models alongside key information, including code and weights as artifacts and configurable parameters that affect model behavior
+- Holistically track all performance metrics related to your logged models with links to training and evaluation jobs
+- Search for and rank existing models based on a set of certain criteria, such as performance metrics on training or evaluation datasets
+
+### Quickstart
+
+This quickstart example demonstrates how to use the LoggedModel lineage feature in MLflow 3.0 with openai sdk, showcasing its integration with runs and traces.
+
+Prerequisite: ``pip install mlflow==3.0.0.rc0 openai``
+
+```python
+from openai import OpenAI
+
+import mlflow
+from mlflow.metrics.genai import answer_correctness, answer_similarity, faithfulness
+
+# turn on autologging for automatic tracing
+mlflow.openai.autolog()
+
+client = OpenAI()
+
+mlflow_ground_truth = (
+    "MLflow is an open-source platform for managing "
+    "the end-to-end machine learning (ML) lifecycle. It was developed by Databricks, "
+    "a company that specializes in big data and machine learning solutions. MLflow is "
+    "designed to address the challenges that data scientists and machine learning "
+    "engineers face when developing, training, and deploying machine learning models."
+)
+# Define evaluation metrics
+metrics = {
+    "answer_similarity": answer_similarity(model="openai:/gpt-4o"),
+    "answer_correctness": answer_correctness(model="openai:/gpt-4o"),
+    "faithfulness": faithfulness(model="openai:/gpt-4o"),
+}
+question = "What is MLflow?"
+
+with mlflow.start_run():
```

Comment:
Can we explain why we're starting a run in the comments? E.g. because we're running an evaluation?